test/helpers/types.ts

```diff
@@ -4,6 +4,18 @@ import { TestERC20 } from '../../typechain'
 export module HelperTypes {
   export type CommandFunction<Input, Output> = (input: Input) => Promise<Output>
 
+  export type Incentive = {
```

Comment:
These should probably go in `contractParams.ts` or elsewhere - these type definitions here are only for the helper commands

Reply:
woops sorry didn't notice that file